test/parallel/test-child-process-detached.js

```diff
@@ -2,24 +2,33 @@
 const common = require('../common');
 const assert = require('assert');
 const path = require('path');
-
 const spawn = require('child_process').spawn;
+const execFile = require('child_process').execFile;
 const childPath = path.join(common.fixturesDir,
                             'parent-process-nonpersistent.js');
 let persistentPid = -1;
+let execPersistentPid = -1;
 
 const child = spawn(process.execPath, [ childPath ]);
+const execChild = execFile(process.execPath, [ childPath ]);
```

Comment:
I don't think this actually tests `detached` being passed to `execFile()`. It doesn't happen here, and the fixture calls `spawn()`.